tests/utils_tests/test_http.py

```diff
@@ -223,6 +223,16 @@ def test_parsing_rfc850(self):
         parsed = http.parse_http_date('Sunday, 06-Nov-94 08:49:37 GMT')
         self.assertEqual(datetime.utcfromtimestamp(parsed), datetime(1994, 11, 6, 8, 49, 37))
 
+    def test_parsing_rfc850_year(self):
+        dates = (
+            ('Monday, 15-Aug-51 08:49:37 GMT', datetime(2051, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-70 00:00:00 GMT', datetime(1970, 8, 15, 0, 00, 00)),
+        )
+        for rfc850str, rfc850date in dates:
+            with self.subTest(string=rfc850str, date=rfc850date):
+                parsed = http.parse_http_date(rfc850str)
```

Comment:
Please rebase your branch on master and remove `http.` after 0e212a705e6b2e49a246b16286036c40ec2ac4f8.